### PR TITLE
tiptap story

### PR DESCRIPTION
**Pull request by justame**

### CODE REVIEW

**Comment by justame** [comment]
@lxgreen I think that the bundle test failed because of this PR that @shaulgo worked on
https://github.com/wix/ricos/commit/39bd91b59c347db641135a6c528f621a7fa85cfb#diff-d75e3690970d498937abfbfdaa89fcc4e035e923bd17bd63f4d1ef7fff639926

**Comment by shaulgo** [comment]
> @lxgreen I think that the bundle test failed because of this PR that @shaulgo worked on
> https://github.com/wix/ricos/commit/39bd91b59c347db641135a6c528f621a7fa85cfb#diff-d75e3690970d498937abfbfdaa89fcc4e035e923bd17bd63f4d1ef7fff639926
> 

The bundle size test is a bit broken. We need to fix it.

File: packages/tiptap-editor/web/.nmprc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry="https://registry.npmjs.org/"
```
Comment (lxgreen) [inline packages/tiptap-editor/web/.nmprc:1]: this does nothing -- file name is invalid. we have root .npmrc, this can be removed

File: packages/tiptap-editor/web/package.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
+{
+    "name": "wix-tiptap-editor",
+    "version": "8.32.0",
+    "author": {
+      "name": "Bashar Qaddumi",
```
Comment (lxgreen) [inline packages/tiptap-editor/web/package.json:6]: yaron, shall we call you Bashar?

File: tsconfig.json [hunk 3]
```diff
@@ -19,6 +19,6 @@
     "allowJs": true
   },
   "files": ["external-types.d.ts"],
-  "include": ["packages", "packages/**/*.json", "*.ts"],
-  "exclude": ["node_modules", "dist"]
+  "include": ["packages/**/*.ts","packages/**/*.tsx", "packages/**/*.json", "*.ts"],
```
Comment (lxgreen) [inline tsconfig.json:22]: why did you need this?
